client/src/components/CreateMemberInfoInput.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+const CreateTextareaContainer = styled.div`
+  display: flex;
+  flex-flow: row wrap;
+  background-color: ${(props) => props.theme.colors.input};
+  border-radius: 20px;
+  border: none;
+  padding: 16px;
+  outline: none;
+  box-shadow: 0px 3px 6px;
+`;
+
+const CreateHeadline = styled.input`
+background-color: ${(props) => props.theme.colors.input};
+color: ${(props) => props.theme.colors.textSecondary}
```

Comment:
The formatting of this file is not working correctly. I guess it's related to a missing semicolon.

```suggestion
color: ${(props) => props.theme.colors.textSecondary};
```

Reply:
Oh yes!